infra/test-buck2.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -xe
+shopt -s expand_aliases
```

Comment:
You could have also used a function FYI

Reply:
Thanks. I will consider that option in the general case. In this case I think using a function might be too much, but if the script starts getting more complex as we add more samples I will switch to using a function.